hieradata/class/integration/mongo.yaml

```diff
@@ -0,0 +1,7 @@
+---
+
+mongodb::s3backup::backup::s3_bucket: govuk-mongodb-backup-s3-integration
+mongodb::s3backup::backup::standalone: 'True'
```

Comment:
Correct again, it should be False. However, this is actually a string and not a boolean, because it is inserted into a python script as a python boolean. In Python, booleans are capitalised.